LYNX-155: Update documentation of queries/mutations to include introduces custom_attributes fields (#136)
AdobeDocs/commerce-webapi: src/pages/graphql/schema/customer/queries/customer.md

}
```

### Retrieve custom attributes metadata from a customer

> The following call returns custom attributes for the logged-in customer. Provide the customer's token in the header section of the query.

Reviewer: Add blank line below.

**Request:**

```graphql
{
  customer {
    firstname
    lastname
    suffix
    email
    custom_attributes {
      code
      ... on AttributeValue {
        value
      }
      ... on AttributeSelectedOptions {
        selected_options {
          label
          value
        }
      }
    }
  }
}
```

**Response:**

```json
{
  "data": {
    "customer": {
      "firstname": "John",
      "lastname": "Doe",
      "suffix": null,
      "email": "jdoe@example.com",
      "custom_attributes": [
        {
          "code": "reward_update_notification",
          "value": "0"
        }
      ]
    }
  }
}
```

### Retrieve custom attributes metadata from a customer address

> The following call returns the customer address custom attributes for the logged-in customer. Provide the customer's token in the header section of the query.

Reviewer: Add blank line below

<CompareListOutput />

> ### AttributeValueInterface attributes

Reviewer: Add blank line below

Author: there is actually a blank line below. no need for an extra line.

The `AttributeValueInterface` contains the following attributes:

Attribute |  Data Type | Description
--- | --- | ---
`uid` | ID! | The unique ID of an attribute value
`code` | String! | The attribute code

> Currently, `AttributeValueInterface` has 2 different implementations: `AttributeValue` and `AttributeSelectedOptions`.

Reviewer: ```suggestion
Currently, `AttributeValueInterface` has two different implementations: `AttributeValue` and `AttributeSelectedOptions`.
```

> Apart from the attributes described for `AttributeValueInterface`, the `AttributeValue` contains the following:

Reviewer: ```suggestion
In addition to the attributes described for `AttributeValueInterface`, the `AttributeValue` contains the following:
```

Attribute |  Data Type | Description
--- | --- | ---
`value` | String! | The attribute value

> On the other hand, the `AttributeSelectedOptions` contains the following attributes:

Reviewer: ```suggestion
The `AttributeSelectedOptions` object contains the following attributes:
```

`country_code` | CountryCodeEnum | The customer's country
`country_id` | String | Deprecated. Use `country_code` instead. The customer's country
`custom_attributes` | [CustomerAddressAttribute](#customeraddressattribute-attributes) | Deprecated. Not applicable for GraphQL

> `custom_attributesV2` | [AttributeValueInterface](#attributevalueinterface-attributes) | Customer address' custom attributes

Reviewer: ```suggestion
`custom_attributesV2` | [AttributeValueInterface](#attributevalueinterface-attributes) | Custom attributes assigned to the customer address
```
